bundles/org.openhab.core.addon/src/test/java/org/openhab/core/addon/AddonInfoListReaderTest.java

```diff
@@ -91,7 +93,13 @@ void testAddonInfoListReader() {
         AddonDiscoveryMethod method = discoveryMethods.get(0);
         assertNotNull(method);
         assertEquals("mdns", method.getServiceType());
-        assertEquals("_printer._tcp.local.", method.getMdnsServiceType());
+        List<AddonParameter> parameters = method.getParameters();
+        assertNotNull(parameters);
+        assertEquals(1, parameters.size());
+        AddonParameter parameter = parameters.get(0);
+        assertNotNull(parameter);
+        assertEquals("mdnsServiceType", parameter.getName());
```

Comment:
"mdnsServiceType" should be a constant

Reply:
Left as is, as this would also require a local constant and the exact name of the parameter is not relevant for the test.